docs/embroider-css-modules-temporary/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "docs-app-for-embroider-css-modules-temporary",
-  "version": "0.1.0",
+  "version": "0.1.2",
```

Comment:
I let the initial release of `docs-app-for-embroider-css-modules-temporary` be `0.1.2` so that the version matches that of `docs-app-for-embroider-css-modules`.